lib/write.go

```diff
@@ -72,6 +72,14 @@ func (a *ACBuild) Write(output string, overwrite, sign bool, gpgflags []string)
 	}
 	defer ofile.Close()
 
+	defer func() {
+		// When write is done, if an error is encountered remove the partial
+		// ACI that had been written.
+		if err != nil {
+			os.RemoveAll(output)
```

Comment:
Note this is redundant with 90/98 (and unclear why inconsistent, RemoveAll here and Remove there)